clap_derive/src/derives/args.rs

```diff
@@ -361,7 +361,7 @@ pub fn gen_augment(
 
 fn gen_arg_enum_possible_values(ty: &Type) -> TokenStream {
     quote_spanned! { ty.span()=>
-        .possible_values(<#ty as clap::ArgEnum>::VARIANTS)
+        .possible_values(<#ty as clap::ArgEnum>::VARIANTS.into_iter().map(|v| *v))
```

Comment:
Scrap my comment about implementing `IntoIterator` all we need is this. 

We still would have the breaking change tho, but that is something I cannot really judge the severity of.

Reply:
well should probably use [`copied`](https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.copied) instead of map. We could then just derive Copy for ArgValue, as that is just 2 references and a bool.